doc/samples/sid_end_device.rst

```diff
@@ -122,88 +90,59 @@ For example:
+==========================
 
-* ``CONFIG_SID_END_DEVICE_AUTO_START`` -- Enables an automatic Sidewalk initialization and start.
+You can choose a sample variant depending on your needs.
+Testing application scenarios differ depending on the chosen Sidewalk configuration:
 
-* ``CONFIG_SID_END_DEVICE_AUTO_CONN_REQ`` -- Enables an automatic connection request before sending a message. 
-  If needed, the Bluetooth LE connection request is sent automatically.
+* ``overlay-hello`` -- The default configuration.
+  Use it to run the Hello Sidewalk variant.
+* ``overlay-demo`` -- Use this configuration to run the Sensor monitoring variant.
+* ``overlay-dut`` -- Use this configuration to run the Device under test (DUT) variant.
+  The overlay enables CLI and disables sample automation.
 
-You can build the ``sensor_monitoring`` end device application with Bluetooth LE only libraries by running the following command in the project directory:
+You can build an end device application with the selected overlay by running the following command in the project directory:
 
 .. parsed-literal::
    :class: highlight
 
-   $ west build -b *build_target* -- -DCONFIG_SID_END_DEVICE_SENSOR_MONITORING=y -DCONFIG_SIDEWALK_SUBGHZ_SUPPORT=n
+   $ west build -b *build_target* -- -DOVERLAY_CONFIG="*overlay-name*.conf"
 
-For example:
+For example, to build the Sidewalk DUT configuration overlay, run the following command:
 
 .. code-block:: console
 
-   $ west build -b nrf5340dk_nrf5340_cpuapp -- -DCONFIG_SID_END_DEVICE_SENSOR_MONITORING=y -DCONFIG_SIDEWALK_SUBGHZ_SUPPORT=n
-
-Source file setup
-*****************
-
-The application consists of two source files:
+   $ west build -b nrf52840dk_nrf52840 -- -DOVERLAY_CONFIG="overlay-dut.conf"
 
-* :file:`main.c` -- The main source file. Starts Sidewalk application.
-* :file:`app.c` (:file:`app.h`) -- The application specific file.
-  It communicates to the sidewalk thread using events defined in :file:`sidewalk.h`, and receives Sidewalk callbacks defined in :file:`sid_api.h`.
-  An application may consist of multiple `app_*` files, to define user interface and realize application logic.
-* :file:`sidewalk.c` (:file:`sidewalk.h`) -- Sidewalk thread file.
-  This is a common source file for all sample configuration.
-  It implements a separate thread where the Sidewalk API is called.
-  The file also includes the support for the DFU state.
-  See the behavior of the state machine outlined in the following diagram:
+See the details on running and testing individual variants, as well as applying available Kconfig options:
 
-   .. uml::
-      :caption: Sidewalk thread state machine
+.. toctree::
+   :maxdepth: 1
+   :glob:
 
-      [*] --> Sidewalk
-      Sidewalk --> [*] : event_factory_reset
-      Sidewalk --> FirmwareUpdate : event_dfu
-      FirmwareUpdate --> Sidewalk : event_dfu
+   variants/hello.rst
+   variants/sensor_monitoring.rst
+   variants/dut.rst
 
 .. _registering_sidewalk:
 
 Registering Sidewalk Endpoint

```

Comment:
Probably should've caught that earlier, just making sure my understanding is correct:
Is "registering sidewalk endpoints" applicable to all variants? E.g., in Hello and Sensor monitoring there's a line:
'Wait for the device to complete the automatic registration'. Is it about the same process?
If so then I'm not sure if the current placement of the "registering sidewalk endpoints" is the best. 
In addition, in Hello Sidewalk, in line 120 this is repeated again as
"Wait for the device to register, or perform registration manually as described in the :ref:`registering_sidewalk` documentation."
I think we should unify how and where this is referenced.